c2rust-analyze/src/borrowck/type_check.rs

```diff
@@ -348,7 +348,7 @@ impl<'tcx> TypeChecker<'tcx, '_> {
             StatementKind::Assign(ref x) => {
                 let (pl, ref rv) = **x;
 
-                if self.special_casts.is_special(&pl, &rv) {
+                if self.c_void_casts.is_special(&pl, &rv) {
```

Comment:
Should it still be `is_special` when it's `c_void_casts` now?

Reply:
```suggestion
                if self.c_void_casts.is_special(&pl, rv) {
```

Reply:
e2b16ed3